### PR TITLE
update kube dependencies to v1.25.0

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

> /kind cleanup

-->
```release-note
This release update kubernetes module dependencies to v1.25
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,77 +1,124 @@
...
-	github.com/google/gofuzz v1.2.0 // indirect
-	github.com/imdario/mergo v0.3.12 // indirect
 	github.com/kubernetes-csi/csi-lib-utils v0.11.0
 	github.com/kubernetes-csi/csi-test/v4 v4.0.2
-	google.golang.org/grpc v1.40.0
+	google.golang.org/grpc v1.47.0
+	k8s.io/api v0.25.0
+	k8s.io/apimachinery v0.25.0
+	k8s.io/client-go v0.25.0
+	k8s.io/csi-translation-lib v0.23.6
```
Comment (jsafrane) [inline go.mod:17]: This should be v0.25.0 too.

Comment (humblec) [reply go.mod]: sure, done @jsafrane

File: go.mod [hunk 2]
```diff
@@ -1,77 +1,124 @@
...
 )
 
-replace k8s.io/api => k8s.io/api v0.24.0
+replace k8s.io/api => k8s.io/api v0.25.0
 
-replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.24.0
+replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.25.0
 
-replace k8s.io/apimachinery => k8s.io/apimachinery v0.24.0
+replace k8s.io/apimachinery => k8s.io/apimachinery v0.26.0-alpha.0
```
Comment (jsafrane) [inline go.mod:76]: Where this comes from? I would expect v0.25.0.

Comment (humblec) [reply go.mod]: @jsafrane i have made the change to replace it with v25. ptal.. thanks.

**Pull request merged**